contracts/test/TestPaymasterPreconfiguredApproval.sol

```diff
@@ -13,12 +13,12 @@ contract TestPaymasterPreconfiguredApproval is TestPaymasterEverythingAccepted {
     }
 
     function acceptRelayedCall(
-        ISignatureVerifier.RelayRequest calldata relayRequest,
-        bytes calldata signature,
-        bytes calldata approvalData,
+        ISignatureVerifier.RelayRequest memory relayRequest,
+        bytes memory signature,
+        bytes memory approvalData,
         uint256 maxPossibleGas
     )
-    external
+    public
```

Comment:
Why?

Reply:
solc abiv2...
(I'll check if solc0.6.9 can compile, and if so, I'll revert it)

Reply:
fixed

Reply:
No it's not fixed. ARC should be external.

Reply:
right (I fixed the code as I wrote the comments, but then had to revert since it didn't compile)